frontend/src/app/app.test.tsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-import Index from '@app/index';
+import Search from '@app/search';
+import "isomorphic-fetch"
 
 import { mount, shallow } from 'enzyme';
 import { Label } from '@patternfly/react-core';
 
 describe('App tests', () => {
```

Comment:
@jmarrero I just saw that we have tests in the react code :open_mouth: :+1: ... how do we run them? I'd like to get them incorporated into our CI build and even our coverage report if possible.

Reply:
We run it by running "yarn test" on the frontend folder. 

adding:

```
      <execution>
            <id>yarn-test</id>
            <goals>
                <goal>yarn</goal>
            </goals>
            <configuration>
                <arguments>test</arguments>
            </configuration>
        </execution>
```

to line 192 of the pom.xml should be enough to run it as part of the build.

Reply:
Let me know if you want me to add this on this PR or if you wanted to do the yarn test in the CI script

Reply:
I don't want to increase the scope of the PR, so lets tackle this separately.
Thanks for the info on how to run it. I will take a look at integrating it with our CI builds.